action.yml

```diff
@@ -43,53 +43,54 @@ runs:
 
         echo "Building with Go ${GO_VERSION}"
         echo "go-version=${GO_VERSION}" >> "${GITHUB_OUTPUT}"
-        GO_MINOR_VERSION=${GO_VERSION%.*}
-        GO_VERSION_PREVIOUS="${GO_MINOR_VERSION%.*}.$((${GO_MINOR_VERSION#*.}-1))"
-        echo "Previous version ${GO_VERSION_PREVIOUS}"
+        GO_MAJOR_VERSION=${GO_VERSION%.*}
+        echo "go-major-version=${GO_MAJOR_VERSION}" >> "${GITHUB_OUTPUT}"
+        GO_MAJOR_VERSION_PREVIOUS="${GO_MAJOR_VERSION%.*}.$((${GO_MAJOR_VERSION#*.}-1))"
+        echo "Previous version: ${GO_MAJOR_VERSION_PREVIOUS}"
 
-    - name: 'Check if Go version is an LTS release'
-      id: check-lts
+    - name: 'Check if Go version is a stable release'
+      id: check-stable
       shell: bash
       run: |
         GO_VERSION=${{ steps.get-go-version.outputs.go-version }}
-        GO_MINOR_VERSION=${GO_VERSION%.*}
+        GO_MAJOR_VERSION=${{ steps.get-go-version.outputs.go-major-version }}
         versions=$(curl -s 'https://api.github.com/repos/golang/go/git/refs/tags' | jq -r '.[].ref' | grep -o 'refs/tags/go[0-9\.]*' | grep -E 'go[0-9]+\.[0-9]+\.[1-9][0-9]*$' | sed 's_refs/tags/__')
         latest_versions=$(echo "$versions" | grep -o 'go[0-9]\+\.[0-9]\+' | sed 's/go//' | sort -uV | tail -2)
         
-        lts=false
+        stable=false
         needs_bump=false
         for version in $latest_versions; do
           echo "$versions" | grep "$version"
-          if [ "$version" = "$GO_MINOR_VERSION" ]; then
-            echo "$GO_VERSION is part of an LTS release of $GO_MINOR_VERSION"
-            lts=true
-            echo "lts=${lts}" >> "${GITHUB_OUTPUT}"
+          if [ "$version" = "$GO_MAJOR_VERSION" ]; then
+            echo "$GO_VERSION is a minor release of $GO_MAJOR_VERSION"
+            stable=true
+            echo "stable=${stable}" >> "${GITHUB_OUTPUT}"
             break
           fi
         done
-        if [ "$lts" = "false" ]; then
-          echo "$GO_VERSION is NOT part of an LTS release of $GO_MINOR_VERSION"
+        if [ "$stable" = "false" ]; then
+          echo "$GO_VERSION is NOT a minor release of $GO_MAJOR_VERSION"
           exit 1
         fi
 
-        latest_patch_version=$(echo "$versions" | grep "$GO_MINOR_VERSION"| sed 's/go//' | sort -V | tail -1)
+        latest_patch_version=$(echo "$versions" | grep "$GO_MAJOR_VERSION"| sed 's/go//' | sort -V | tail -1)
         if [ "$latest_patch_version" != "$GO_VERSION" ]; then
-          echo "$GO_VERSION needs to be bumped to $latest_patch_version"
+          echo "$GO_VERSION can be bumped to $latest_patch_version"
```

Comment:
~~ Minor suggestion: I could see enhancements in the future taking into account CVEs, etc., so softening this language might make it clearer to those unfamiliar w/ the tool that this is a proactive bump